cudamapper/src/utils.cpp

```diff
@@ -305,7 +304,11 @@ void print_sam(const std::vector<Overlap>& overlaps,
         // TODO: write AUX data if available
 
         std::lock_guard<std::mutex> lg(write_output_mutex);
-        int out = sam_write1(file.get(), header.get(), alignment.get());
+        result = sam_write1(file.get(), header.get(), alignment.get());
+        if (result < 0)
+        {
+            fprintf(stderr, "print_sam: could not write alignment");
```

Comment:
We use `std::cerr` instead of `fprintf`. It's not ideal, but we should still try to be consistent

Reply:
Fixed everywhere in latest version